atlasdb-client/src/main/java/com/palantir/atlasdb/offheap/rocksdb/RocksDbPersistentTimestampStore.java

```diff
@@ -80,6 +123,13 @@ public void put(StoreNamespace storeNamespace, @Nonnull Long startTs, @Nonnull L
         putEntry(availableColumnFamilies.get(storeNamespace.uniqueName()), key, value);
     }
 
+    @Override
+    public void multiPut(StoreNamespace storeNamespace, Set<Map.Entry<Long, Long>> toWrite) {
+        for (Map.Entry<Long, Long> entry : toWrite) {
```

Comment:
as above, it may be easier to pass in `Map<Long, Long>`

Reply:
Changed.